wgpu-core/src/command/render.rs

```diff
@@ -1346,6 +1346,15 @@ impl Global {
             let query_sets = hub.query_sets.read();
             let texture_views = hub.texture_views.read();
 
+            if desc.color_attachments.len() > hal::MAX_COLOR_ATTACHMENTS {
```

Comment:
Should this check use limits from device?

Reply:
It should.